### PR TITLE
Playfair: Version 2.000;gftools[0.9.27] added

### PR DESCRIPTION
**Pull request by emmamarichal**
1a50909: [gftools-packager] Playfair: Version 2.000;gftools[0.9.27] added

* Playfair Version 2.000;gftools[0.9.27] taken from the upstream repo https://github.com/googlefonts/Playfair at commit https://github.com/googlefonts/Playfair/commit/1ef0096ddd847f51974f3a956c572f37e00b04a6.

 e7a2aa7: Playfair: description and metadata updated

### CODE REVIEW

**Comment by emmamarichal** [comment]
Updated

## Playfair: Version 2.000;gftools[0.9.27] added

---
 f510614: [gftools-packager] Playfair: Version 2.000;gftools[0.9.27] added

* Playfair Version 2.000;gftools[0.9.27] taken from the upstream repo https://github.com/googlefonts/Playfair at commit https://github.com/googlefonts/Playfair/commit/19dc2fbe72f9897b277c0d1044423d8dd089ab73.

 a4d49c0: Playfair: description and metadatas

**Comment by emmamarichal** [comment]
Those fails are here mainly because:
- There is no Regular master with the following values: opsz = 12pt / wght = 400 / wdth = 100
- There is a RFN (Dave is checking this)
- The optical size values don’t match the specs (we decided to keep the author ones)

**Comment by emmamarichal** [comment]
Dave has to check for the RFN

**Comment by emmamarichal** [comment]
Updated

## Playfair: Version 2.000;gftools[0.9.28] added

---
 18bcdf5: [gftools-packager] Playfair: Version 2.000;gftools[0.9.28] added

* Playfair Version 2.000;gftools[0.9.28] taken from the upstream repo https://github.com/googlefonts/Playfair at commit https://github.com/googlefonts/Playfair/commit/834cc36f15b71db05459d4edeaa9bce594ea1863.

 2ffc728: Updated metadatas and description

**Comment by emmamarichal** [comment]
@davelab6 @RosaWagner @m4rc1e @chrissimpkins 
I removed the RFN and re-packaged the fonts. Should be ready now!

File: ofl/playfair/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  copyright: "Copyright 2005 The Playfair Project Authors (https://github.com/clauseggers/Playfair)"
+}
+fonts {
+  name: "Playfair"
+  style: "italic"
+  weight: 400
+  filename: "Playfair-Italic[opsz,wdth,wght].ttf"
+  post_script_name: "Playfair-5ptSemiExpandedLightItalic"
+  full_name: "Playfair 5pt SemiExpanded Light Italic"
+  copyright: "Copyright 2005 The Playfair Project Authors (https://github.com/clauseggers/Playfair)"
```
Comment (RosaWagner) [inline ofl/playfair/METADATA.pb:23]: copyright string should be the same everywhere (repo from googlefonts)

File: ofl/playfair/OFL.txt [hunk 2]
```diff
@@ -0,0 +1,93 @@
+Copyright 2005 The Playfair Project Authors (https://github.com/clauseggers/Playfair)
```
Comment (RosaWagner) [inline ofl/playfair/OFL.txt:1]: copyright string should be the same everywhere (repo from googlefonts)

**Comment by RosaWagner** [comment]
🔥 FAIL: The variable font 'wdth' (Width) axis coordinate must be 100 on the 'Regular' instance. ([com.google.fonts/check/varfont/regular_wdth_coord](https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/fvar.html#com.google.fonts/check/varfont/regular_wdth_coord))

-> The default location can be SemiExpanded, but the instances should have normal width (100)

Capture d’écran 2023-04-12 à 12 33 26

**Comment by emmamarichal** [comment]
Updated

## Playfair: Version 2.000;gftools[0.9.28] added

---
 ebcc99c: [gftools-packager] Playfair: Version 2.000;gftools[0.9.28] added

* Playfair Version 2.000;gftools[0.9.28] taken from the upstream repo https://github.com/googlefonts/Playfair at commit https://github.com/googlefonts/Playfair/commit/429643a98e2cad452f9fbc103680d17b1e0a1c50.

 7400327: Description / Metadata

**Comment by emmamarichal** [comment]
I'm trying to solve this fail, but I don't understand why it appears now. I just changed the instances values to get 12pt - 100 in the Fvar, instead of 5pt - 112,5.
🔥 FAIL: Checking OS/2 usWeightClass. ([com.google.fonts/check/usweightclass](https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/usweightclass))
🔥 FAIL OS/2 usWeightClass is '300' when it should be '400'. [code: bad-value]

The weightclass is indeed 300 because the variable font origin is Light (we don't have regular master), but why the fail doesn't appear on the Regular? 🤔
Capture d’écran 2023-04-13 à 11 36 26

**Comment by emmamarichal** [comment]
Updated

## Playfair: Version 2.000;gftools[0.9.28] added

---
 a7e21fe: [gftools-packager] Playfair: Version 2.000;gftools[0.9.28] added

* Playfair Version 2.000;gftools[0.9.28] taken from the upstream repo https://github.com/googlefonts/Playfair at commit https://github.com/googlefonts/Playfair/commit/209226cb60e266cbe061819e2336ed5c4259f026.

 f6887c0: Description/metadata updated

**Comment by RosaWagner** [comment]
The fail might be a bug from Fontbakery which used to not accept font origin different from 400. It seems that it has been fixed for Roman style, but not italic. Can you open an issue in Fontbakery and linking this PR?

**Pull request merged**